.github/workflows/production.yaml

```
@@ -30,8 +30,15 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runnner.os }}-yarn-${{ hashFiles('**/yarn.lock.json') }}
```

```suggestion
          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock.json') }}
```

@flat35hd99 I found the spell miss.